hackathon_site/dashboard/frontend/src/pages/TeamDetail/TeamDetail.tsx

```diff
@@ -0,0 +1,106 @@
+import React from "react";
+import Header from "components/general/Header/Header";
+import Typography from "@material-ui/core/Typography";
+import { mockTeamMultiple } from "testing/mockData";
+import {
+    Table,
+    TableBody,
+    TableContainer,
+    TableHead,
+    TableRow,
+    TableCell,
+    Checkbox,
+    Paper,
+    Container,
+    Grid,
+} from "@material-ui/core";
+
+const TeamDetail = () => {
+    return (
+        <>
+            <Header />
+            <Container>
+                <Grid container direction="column" spacing={6}>
+                    <Grid item>
+                        <Typography variant="h1">
+                            Team {mockTeamMultiple.id} Overview
+                        </Typography>
+                    </Grid>
+                    {/* Team info and actions */}
+                    <Grid container spacing={2}>
+                        <Grid
```

Comment:
Could you create a new component for the team info table (it can include the grid elements too). The reason why is because this file is getting really big and we're not even done have the page yet 😅 to avoid bloated code, create a component in
`src/components/teamDetail/TeamInfoTable` with 2 files:
`TeamInfoTable.tsx` - you can name it whatever you want, just make sure it matches the folder name
`TeamInfoTable.test.tsx` - move your tests that only test what's displayed on the table to here
you don't have custom styling so you don't need to create a scss file.
Then, place the component here in TeamDetail.tsx.